docs/guides/pay_per_event.mdx

```diff
@@ -0,0 +1,37 @@
+---
+id: pay-per-event
```

Comment:
since we shipped this as v3.3, we should update the v3.3 docs snapshot too

(but lets keep that as the final step before merging for easier review)

Reply:
How does one go about that?

Reply:
for API docs: inside the website folder just run `npx docusaurus api:version 3.3`, maybe you'll need to remove the JSON file first, but I don't think its necessary

the regular docs are done the same way via `npx docusaurus docs:version 3.3`, but that generates a lot of files and touches the versions.json, so you would have to first wipe all of that. 

---

anyway, i just found out i forgot to do this, and we are missing the v3.3 snapshot completely :D so once we merge this, it's as simple as calling those and committing what it generates

```
cd website
npx docusaurus docs:version 3.3
npx docusaurus api:version 3.3
```

Reply:
also we'll need to reindex algolia, so ping me once its done

(another thing we should ideally automate)